src/ensembl/src/shared/helpers/numberFormatter.ts

```diff
@@ -2,6 +2,6 @@
     Formats the input number to comma separated representation
     eg: 10000 -> 10,000
 */
-export const getCommaSeparatedNumber = (input: number): string => {
+export const getCommaSeparatedNumber = (input: number | string): string => {
```

Comment:
I am not sure extending the type of the input to `number | string` is a good thing. With input type as number we are guaranteed that whatever is passed to this function is actually a number. With allowing a string to be passed, we no longer have this guarantee.

Reply:
I have now made changes to `getCommaSeparatedNumber` to accept only `number` as before. There weren't any other changes required for the coordinates as all of them are of type `number`.